Essentials/src/com/earth2me/essentials/Console.java

```diff
@@ -2,15 +2,17 @@
 
 import com.earth2me.essentials.messaging.IMessageRecipient;
 import com.earth2me.essentials.messaging.SimpleMessageRecipient;
-
 import org.bukkit.Server;
 import org.bukkit.command.CommandSender;
 
+import static com.earth2me.essentials.I18n.tl;
+
 
 public final class Console implements IMessageRecipient {
     public static final String NAME = "Console";
+    public static final String DISPLAY_NAME = tl("consoleName");
     private static Console instance; // Set in essentials
-    
+
```

Comment:
Unnecessary diff

Reply:
Auto formatting. There was 4 black spaces on this empty line.